std/json.d

```diff
@@ -73,6 +73,7 @@ Flags that control how json is encoded and parsed.
 enum JSONOptions {
     none,                       /// standard parsing
     specialFloatLiterals = 0x1, /// encode NaN and Inf float values as strings
+    escapeMBC = 0x2             /// encode multi byte characters using an unicode escape sequence
```

Comment:
I'm not sure I like the name `escapeMBC`... Maybe `escapeMultibyteChar`? I know it's a bit more verbose, but the acronym MBC made no sense to me until I read the docs.